tests/utils/utils.py

```diff
@@ -1,3 +1,35 @@
+from starkware.cairo.lang.tracer.profile import profile_from_tracer_data
+from starkware.cairo.lang.tracer.tracer_data import TracerData
+from starkware.starknet.business_logic.execution.execute_entry_point import (
+    ExecuteEntryPoint,
+)
+
+# Override _run() entrypoint for hooking and profiling cairo runner.
+old_run = ExecuteEntryPoint._run
+
+
+def override_run(*args, **kwargs):
+    runner, syscall_handler = old_run(*args, **kwargs)
+
+    if traceit._context:
+        runner.relocate()
+
+        tracer_data = TracerData(
+            program=runner.program,
+            memory=runner.relocated_memory,
+            trace=runner.relocated_trace,
+            program_base=1,
+            debug_info=runner.get_relocated_debug_info(),
+        )
+
+        profile = profile_from_tracer_data(tracer_data)
+        _profile_datas[traceit._context] = profile
+    return runner, syscall_handler
+
+
+ExecuteEntryPoint._run = override_run
```

Comment:
can you move this after the import statements? and don't forget to run `make format` to make sure the ordering is the right one